src/main/scala/uk/gov/nationalarchives/draftmetadatavalidator/Lambda.scala

```diff
@@ -138,6 +140,11 @@ class Lambda {
     filteredMetadata.map(_.name)
   }
 
+  private def cleanupRootDirectory() = {
+    val directory = new Directory(new File(rootDirectory))
```

Comment:
Have you checked that for each Lambda instance it has a different root directory and the Lambdas do not share one? Otherwise this might have the effect of deleting a concurrently running Lambda's csv file?

Reply:
Could you save the file in `/{root}/{consignment id}` then just clean that directory out?

Reply:
Yeah, we can do this as well as I'm already saving the file in the` /{root}/{consignment id}` folder.